config.example.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 var config = {}
+config.session = {}
```

Comment:
Because you can't set a property on undefined:

```node
👌  hello-domains-node git:(update--to-recent-api-changes) node
> var config = {}
undefined
> config.session.secret = 'somelongrandomstring';
TypeError: Cannot set property 'secret' of undefined
    at repl:1:23
    at realRunInThisContextScript (vm.js:22:35)
    at sigintHandlersWrap (vm.js:98:12)
    at ContextifyScript.Script.runInThisContext (vm.js:24:12)
    at REPLServer.defaultEval (repl.js:346:29)
    at bound (domain.js:280:14)
    at REPLServer.runBound [as eval] (domain.js:293:12)
    at REPLServer.onLine (repl.js:545:10)
    at emitOne (events.js:101:20)
    at REPLServer.emit (events.js:188:7)
>
```